website/docs/docs/deploy/continuous-integration.md

```diff
@@ -27,6 +27,27 @@ When the CI run completes, you can view the run status directly from within the
 
 dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has schema customization using the [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macro, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Troubleshooting](/docs/deploy/ci-jobs#troubleshooting).
 
+## Git providers who support CI
+
+If your git provider has a native dbt Cloud integration, you can set up continuous integration (CI) jobs within dbt Cloud.
+
+| **Git provider** | **Native dbt Cloud integration** |[**Automated CI job**](/docs/deploy/ci-jobs)|**Git import with URL**| **Information**|
```

Comment:
going to link in the intro sentence above as it was looking odd i the table with the orange backgroun

![Screenshot 2024-12-19 at 14 35 27](https://github.com/user-attachments/assets/3d0a8426-5acd-4131-9665-f8f166fc148a)


```suggestion
| **Git provider** | **Native dbt Cloud integration** |**Automated CI job** |**Git import with URL**| **Information**|
```